### ISSUE TITLE
After selecting "Delete" and "Update", no information is getting announced by talkback.

### ISSUE THREAD

**Issue by SuryacharanM** [dotnet/maui#6477]
GitHubTags:
#.NET MAUI; #AILimited; #A11ySev2; #A11yMAS; #A11yTCS; #Android; #AndroidPhone; #WCAG1.3.1; #GH_.NETMAUI_Android_Apr2022; #Info and Relationships​;

### Description

Talkback is not announcing any information or regarding the change in the page when we select "Delete" and "Update" controls.

### Steps to Reproduce

1. Install and open ".NET MAUI" app.
2. Swipe till "My Recipes" and double tap.
3. Swipe till any existing recipe and double tap.
4. "Recipe" Screen will open.
5. Swipe till "Edit" button and double tap.
6. Swipe till "Delete" and "Update" controls and observe the issue.

### Version with bug

https://user-images.githubusercontent.com/86710788/165446989-ca204968-63f0-4a70-b550-762e24a4d400.mp4

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 11

### Relevant log output

```shell
Talkback should announce any information or regarding the change in the page when we select "Delete" and "Update" controls.
```

**Comment by SuryacharanM** [comment]
Verified issue in ".NET MAUI" app. issue is fixed so closing the bug
Android Version: Android 11
Added attachment for references

https://user-images.githubusercontent.com/86710788/178658316-4399f0cc-c3f6-4d62-accc-1517da33e8d2.mp4
